src/osbuild-manifests/platform.live-iso.ipp.yaml

```
@@ -0,0 +1,73 @@
+# This file defines the pipeline for building the live ISO.
+version: '2'
+pipelines:
+  - name: live-iso
+    stages:
+      - type: org.osbuild.copy
+        inputs:
+          tree:
+            type: org.osbuild.tree
+            origin: org.osbuild.pipeline
+            references:
+              - name:metal
+        options:
+          paths:
+            - from:
+                mpp-format-string: "input://tree/{artifact_name_prefix}-metal.{arch}.raw"
+              to: tree:///metal.raw
+      - type: org.osbuild.copy
+        inputs:
+          tree:
+            type: org.osbuild.tree
+            origin: org.osbuild.pipeline
+            references:
+              - name:metal4k
+        options:
+          paths:
+            - from:
+                mpp-format-string: "input://tree/{artifact_name_prefix}-metal4k.{arch}.raw"
+              to: tree:///metal4k.raw
+      # We need to be able to create efiboot.img, a FAT filesystem image. It's
+      # hard to setup loopback devices from within the stage, so just do it
+      # here. This should normally be conditional on the architecture but it
+      # doesn't hurt either since it's a tiny file.
+      - type: org.osbuild.truncate
+        options:
+          filename: efiboot.img
+          size:
+            mpp-format-string: '{live_efiboot_img_size_mb * 1024 * 1024}' 
```

```suggestion
            mpp-format-string: '{16 * 1024 * 1024}' 
```

We discussed this last week and decided to just hardcode it to 16 here for now and add a comment about it and mention we can parameterize it if we ever need to adjust it in the future.